src/test/java/com/google/cloud/genomics/dataflow/functions/AlleleSimilarityCalculatorTest.java

```diff
@@ -0,0 +1,84 @@
+package com.google.cloud.genomics.dataflow.functions;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.api.services.genomics.model.Variant;
+import com.google.cloud.dataflow.sdk.transforms.DoFnTester;
+import com.google.cloud.dataflow.sdk.values.KV;
+import com.google.cloud.genomics.dataflow.utils.DataUtils;
+
+public class AlleleSimilarityCalculatorTest {
```

Comment:
As a general stylistic comment, i usually prefer table-driven tests, where we iterate across a collection and pass each pair of values into a single assertXXX statement in the body of a loop, vs. having unrolled loops where I make N calls to the assert methods. But other people disagree, so ignore if you wish.